vertx-junit5/src/main/java/io/vertx/junit5/VertxTestContext.java

```diff
@@ -79,6 +81,19 @@ public synchronized boolean completed() {
     return !failed() && releaseLatch.getCount() == 0;
   }
 
+  /**
+   * Gives the call sites of all unsatisifed checkpoints.
+   *
+   * @return a list of {@link StackTraceElement} references pointing to the unsatisfied checkpoint call sites.
+   */
+  public List<StackTraceElement> unsatisfiedCheckpointCallSites() {
```

Comment:
If the list order is important when the doc should probably explain what it is. If not, then why is the return type a list?

Reply:
It's a list of `StackTraceElement` where each element points to the line where an unsatisfied flag has been created. The order does not matter, what you really want is some diagnosis like:

```
-> Flag in Foo.java at line 123 was not satisfied
-> Flag in Foo.java at line 456 was not satisfied
```

API implementers may use this API, but users will really just see the diagnosis message in logs through `VertxExtension`.

Reply:
Personally I would use `Collection` or `Iterable`, but I guess `List` is fine.